onset_maproom/calc.py

```diff
@@ -102,50 +99,47 @@ def onset_date(
+        For the following `dry_spell_search` days
     
-        Parameters
-        ----------
-        daily_rain : DataArray
-                    Array of daily rainfall values.
-        wet_thresh : int
-                    Rainfall threshold to determine wet day.
-        wet_spell_length : int
-                    Length in days of running window when 'wet_thresh' is to be met to define a wet spell.
-        wet_spell_thresh : int
-                    Threshold of rainfall to be reached during 'wet_spell_length'
-                    window to define a wet spell.
-        min_wet_days : int
-                    Minimum number of wet days in 'wet_spell_length' window when it rained at or above
-                    'wet_spell_thresh' to be considered a wet spell.
-        dry_spell_length : int
-                    Length in days of dry spell that would invalidate a wet spell as onset date
-                    if found within the 'dry_spell_search' days following the 'wet_spell_length'-day window
-                    that met the wet spell criteria.
-        dry_spell_search : int
-                    Length in days to search for a 'dry_spell_length'-day dry spell after a wet spell
-                    is found that would invalidate the wet spell as onset date.
-        time_coord : str, optional
-                    Time grid in 'soil_moisture' (the default is "T", which implies
-                    naming convention of time grid as "T" in 'soil_moisture')         
-        Returns
-        -------
-        onset_delta : DataArray    
-                    The time delta rom the first day of daily_rain
-                    to the first wet day in that wet spell.
-        See Also
-        --------
-        Notes
-        -----
-        Examples
-        --------
-
+    Parameters
+    ----------
+    daily_rain : DataArray
+        Array of daily rainfall values.
+    wet_thresh : int
+        Rainfall threshold to determine wet day if `daily_rain` is greater than `wet_thresh`.
+    wet_spell_length : int
+        Length in days of running window when `wet_thresh` is to be met to define a wet spell.
+    wet_spell_thresh : int
+        Threshold of rainfall to be reached during `wet_spell_length`
+        window to define a wet spell.
+    min_wet_days : int
+        Minimum number of wet days in `wet_spell_length` window when it rained at or above
+        `wet_spell_thresh` to be considered a wet spell.
+    dry_spell_length : int
+        Length in days of dry spell that would invalidate a wet spell as onset date
+        if found within the `dry_spell_search` days following the `wet_spell_length`-day window
+        that met the wet spell criteria.
+    dry_spell_search : int
+        Length in days to search for a `dry_spell_length`-day dry spell after a wet spell
+        is found that would invalidate the wet spell as onset date.
+    time_coord : str, optional
+        Time grid in `daily_rain` (default time_coord='T').       
```

Comment:
`time_coord`

Reply:
I'm not sure what Rémi had in mind, but I would like to point out that outside of the Ingrid bubble, a "grid" is a multidimensional coordinate system. I don't know how it came to mean a single dimension of said coordinate system in the Ingrid world, and I don't want that misuse to persist in our python libraries.